src/util/invariant.h

```diff
@@ -10,6 +10,8 @@ Author: Martin Brain, martin.brain@diffblue.com
 #define CPROVER_UTIL_INVARIANT_H
 
 #include <stdexcept>
+#include <type_traits>
+#include <iostream>
```

Comment:
We must avoid `#include <iostream>` in a widely used header file to keep compilation times reasonable.